lib/apiservers/engine/backends/volume.go

```diff
@@ -234,7 +242,7 @@ func translateInputsToPortlayerRequestModel(name, driverName string, opts, label
 	model.Metadata = make(map[string]string)
 	model.Metadata[dockerMetadataModelKey] = metadata
 	if err := validateDriverArgs(opts, model); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Bad Driver Arg: %s", err)
```

Comment:
I do no think that it handles the driver args at all. Since those args are specific to the plugin itself. I am not sure what docker returns to be honest. since their `local` default driver does not have any driver args that I am aware off.